task/createPullRequest.ps1

```diff
@@ -20,6 +22,8 @@ function RunTask {
     try {
         # Get inputs
         $sourceBranch = Get-VstsInput -Name 'sourceBranch' -Require
+		$targetRepo = Get-VSTSInput -Name 'targetGitRepositoryId'
+		$targetProject = Get-VSTSInput -Name 'targetProjectId'
```

Comment:
If you don't use the `targetProjectId` in the script, you don't need to get his value here.

Reply:
i am using the targetProject in the script.

Reply:
Oh ok, great! I didn't notice!